lib/unison-sqlite/src/Unison/Sqlite/DataVersion.hs

```diff
@@ -0,0 +1,17 @@
+module Unison.Sqlite.DataVersion
```

Comment:
I don't see this used anywhere in this PR, what's the plan for it?

Reply:
It's to replace a thing in `unison-codebase-sqlite`